### PR TITLE
Kelsonic 484 assistive devices lists nesting

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
I accidentally hit `Squash & Merge` before sending up the updates from the comments on https://github.com/department-of-veterans-affairs/vets-website/pull/10780. This is a follow-up PR for those comments, thanks!

## Testing done
Please refer to: https://github.com/department-of-veterans-affairs/vets-website/pull/10780

## Screenshots
Please refer to: https://github.com/department-of-veterans-affairs/vets-website/pull/10780

## Acceptance criteria
Please refer to: https://github.com/department-of-veterans-affairs/vets-website/pull/10780

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/facility-locator/components/AppointmentInfo.jsx [hunk 1]
```diff
@@ -113,9 +113,9 @@ export default class AppointmentInfo extends Component {
         );
 
       return (
-        <Fragment>
+        <>
```
Comment (kelsonic) [inline src/applications/facility-locator/components/AppointmentInfo.jsx]: This was done in reference to: https://github.com/department-of-veterans-affairs/vets-website/pull/10780#discussion_r326361499 @rianfowler

File: src/applications/facility-locator/components/AppointmentInfo.jsx [hunk 2]
```diff
@@ -113,9 +113,9 @@ export default class AppointmentInfo extends Component {
...
           <li key="specialty-care">Specialty care:</li>
-          <ul className="sublist">
+          <ul className="vads-u-padding-left--0">
```
Comment (kelsonic) [inline src/applications/facility-locator/components/AppointmentInfo.jsx]: This was done in reference to: https://github.com/department-of-veterans-affairs/vets-website/pull/10780#discussion_r326361499 @rianfowler

**Review by ncksllvn — APPROVED** [review]
LGTM!

**Comment by kelsonic** [comment]
Added a commit that ensures that the issue is resolved from validation (https://github.com/department-of-veterans-affairs/va.gov-team/issues/484#issuecomment-533637502). I used the [Axe chrome extension](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd) and it was necessary to nest the sublist `<ul>` tag underneath the `<li>` tag to resolve the issue.

**Comment by kelsonic** [comment]
@department-of-veterans-affairs/frontend-review-group PR is ready for review 👍

**Review by cvalarida — APPROVED** [review]

**Pull request merged**